MQ2PluginManager.cpp

```diff
@@ -94,6 +100,64 @@ void DoPluginTool(PSPAWNINFO pChar, PCHAR szLine)
 	}
 }
 
+void DrawPluginManager_MQSettingsPanel()
+{
+	// Toggle for MQ2PluginManager itself
+	if (ImGui::Selectable("", false))
+	{
+		// Gracefully shutdown MQ2PluginManager
+		ShutdownPlugin();
+		WriteChatf("\atPlugin Manager\ax \arUnloading!\ax");
+		// Trying to close ourself with mq:UnloadPlugin causes a crash, so we issue the command for this one.
+		DoCommand(GetCharInfo()->pSpawn, "/plugin MQ2PluginManager unload");
+		return;
+	}
```

Comment:
Having this at all feels strange to me.

Reply:
- Calling ShutdownPlugin is kinda bad because unloading the plugin will call ShutdownPlugin a 2nd time.

Some other things to note:
- `GetCharInfo()->pSpawn` should not ever be used. Use pLocalPlayer instead.
- DoCommand doesn't need to take a spawn. that overload is deprecated, it does nothing with that parameter.
- clicking this will crash if you're not logged in :)

Reply:
i was trying to avoid it.
using mq::UnloadPlugin() causes a lock / silent crash. 
and just calling ShutdownPlugin() leaves it running but closes all of the windows. 

not sure how to address this but the command seemed to work.

Reply:
1. you can use the command, but note the adjustments you'll need to make to it that I called out above.
2. I don't like this feature, I think people will accidentally click it, and its not like you can click it again to re-enable. I don't recommend leaving this in.

Reply:
My biggest complaint with PluginManager is people learn to use it and not the /plugin command. Big contact point for help is people who accidentally unloaded this and don't know how to get it back. So, doesn't seem like there is any value in having a clickable way to unload this here.

Reply:
I can completely agree with that. 
I will remove the toggle for itself. I will also have it not displaying itself in the table for the same reasons.